data/layout/telescope_positions-North-1LST.ecsv

```diff
@@ -1,55 +1,48 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string}
-# - {name: pos_x, unit: m, datatype: float64}
-# - {name: pos_y, unit: m, datatype: float64}
-# - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
+# - {name: telescope_name, datatype: string, description: common telescope name}
+# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST15 (central MST)}
+# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST15 (central MST)}
+# - {name: pos_z, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
 # meta:
+#   data_type: positionfile
+#   description: "telescope positions for CTA North (La Palma)"
 #   EPSG: 32628
 #   center_alt: !astropy.units.Quantity
 #     unit: &id001 !astropy.units.Unit {unit: m}
-#     value: 2150.0
+#     value: 2177
 #   center_easting: !astropy.units.Quantity
 #     unit: *id001
-#     value: 217654.06411838497
+#     value: 217611.227
```

Comment:
Units of easting and northing is `m`.

Reply:
It is already meter. The pointer points to `#     unit: &id001 !astropy.units.Unit {unit: m}`

Reply:
OK, understand. Seems to me a horrible convention - to use pointers to another place in the file. 

It kills readability completely (as soon as you have a couple of pointers), and readability is the reason why we use ecsv files and not a binary format.